{"uid":"cf47080ae5b2222","name":"test_list_advisors_invalid_content_type","fullName":"tests.advisor.test_list_advisors#test_list_advisors_invalid_content_type","historyId":"8252602bcf8ad7fed297ce5238149989","time":{"start":1729945945289,"stop":1729945948223,"duration":2934},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAttributeError: 'list' object has no attribute 'get'","statusTrace":"get_headers = <function get_headers.<locals>._get_headers at 0x7fb4597b7420>\n\n    @allure.suite('List Advisors')\n    @allure.epic('Advisor')\n    @allure.feature('List Advisors')\n    @allure.story('List Advisors with Invalid Content-Type Header')\n    @allure.tag('author: Jeyson')\n    @pytest.mark.functional\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\",condition=True)\n    def test_list_advisors_invalid_content_type(get_headers):\n        url = AdvisorEndpoints.get_advisors()\n        headers = Auth().auth_valid_credential(get_headers).copy()\n        headers['Content-Type'] = 'application/xml'\n        response = RealHomeRequest.get_with_url_headers(url, headers)\n    \n>       assert_status_code_bad_request(response)\n\ntests/advisor/test_list_advisors.py:178: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ncore/assertions/status_code.py:20: in assert_status_code_bad_request\n    assert response.status_code == 400, f\"Expected status code 400 but got {response.status_code}. Error message: {extract_error_message(response)}\"\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    def extract_error_message(response):\n        try:\n>           error_info = response.json().get('error', {})\nE           AttributeError: 'list' object has no attribute 'get'\n\ncore/assertions/status_code.py:39: AttributeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1729945945288,"stop":1729945945288,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"epic","value":"Advisor"},{"name":"tag","value":"author: Jeyson"},{"name":"story","value":"List Advisors with Invalid Content-Type Header"},{"name":"feature","value":"List Advisors"},{"name":"suite","value":"List Advisors"},{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"negative"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.advisor"},{"name":"host","value":"fv-az1144-789"},{"name":"thread","value":"2089-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.advisor.test_list_advisors"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["negative","functional","author: Jeyson","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"cf47080ae5b2222.json","parameterValues":[]}